app/Http/Requests/BotUpdateRequest.php

```diff
@@ -26,18 +26,27 @@ public function authorize()
     public function rules()
     {
         return [
-            'name' => 'required|max:50',
-            'name' => Rule::unique('bots')->ignore($this->id)->where(function ($query) {
-                return $query->where('user_id', \Auth::id());
-            }),
-            'cw_id' => 'required|max:10',
-            'cw_id' => Rule::unique('bots')->ignore($this->id)->where(function ($query) {
-                return $query->where('user_id', \Auth::id());
-            }),
-            'bot_key' => 'required|max:50',
-            'bot_key' => Rule::unique('bots')->ignore($this->id)->where(function ($query) {
-                return $query->where('user_id', \Auth::id());
-            }),
+            'name' => [
+                'required',
+                'max:50',
+                Rule::unique('bots')->ignore($this->id)->where(function ($query) {
+                    return $query->where('user_id', \Auth::id());
+                })
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Http/Requests/BotUpdateRequest.php</br>**Line**: 34</br>Comma required after last value in array declaration